snprc_ehr/resources/queries/snprc_ehr/ExportSndLookupSets.sql

```diff
@@ -0,0 +1,14 @@
+SELECT
+    SetName as LOOKUP_KEY,
+    ObjectId as OBJECT_ID,
+    LEFT(s.Email,
+    CASE WHEN charindex('@', s.Email) = 0
+    THEN LEN(s.Email)
+    ELSE (charindex('@', s.Email) - 1) END) as USER_NAME,
+    ls.Modified as ENTRY_DATE_TM
+FROM
+    snd.LookupSets ls
+    INNER JOIN
+    core.SiteUsers s
```

Comment:
I could not find core.SiteUsers but i do see core.Users. i could also see core.ActiveUsers but could not find SiteUsers. Please correct me if I am wrong.

Reply:
Table can be viewed in the Query Browser on labkey

Reply:
Ok Jay got it under the Schema. Thanks Jay